dashboard/app/jobs/evaluate_rubric_job.rb

```diff
@@ -290,7 +290,11 @@ def self.get_lesson_s3_name(script_level)
 
   private def read_file_from_s3(lesson_s3_name, key_suffix)
     key = "teaching_assistant/lessons/#{lesson_s3_name}/#{key_suffix}"
-    s3_client.get_object(bucket: S3_AI_BUCKET, key: key)[:body].read
+    if [:development, :test].include?(rack_env) && File.exist?(File.join("local-aws", S3_AI_BUCKET, key))
```

Comment:
This lets me override the system-prompt in S3 with a locally sourced one. Helped me greatly when testing different prompts for line annotations. Thought I might keep it?

Reply:
Added a puts in there explaining what local directory it was reading the prompt from.